test/claim/claim-with-request.testSuite.ts

```diff
@@ -286,4 +289,80 @@ export const claimWithRequestTestSuite = () => {
       requestOrigin
     );
   });
+  it(`should fetch all issued claims by revoker did`, async () => {
+    const [requester, revoker, issuer] = await Promise.all([
+      randomUser(),
+      randomUser(),
+      randomUser(),
+    ]);
+    await createRole(
+      {
+        name: 'testrevoke',
+        issuerDid: [issuer.wallet.address],
+        revokerDid: [revoker.wallet.address],
+        ownerAddr: issuer.wallet.address,
+      },
+      roleService
+    );
+    const claimData = await createClaimRequest(
+      'testrevoke.roles.e2e.iam.ewc',
+      requester,
+      issuer,
+      201
+    );
+    await issueClaimRequest(claimData, issuer, 201);
+
+    const result = await claimService.getByRevoker({
+      currentUser: revoker.did,
+    });
```

Comment:
We should make here an actual request to controller using `supertest` library

Reply:
@Harasz Do you know how to mock the @User param with supertest?

Reply:
Oh  awesome, thank you!